x-pack/metricbeat/module/azure/compute_vm/_meta/docs.asciidoc

```diff
@@ -1 +1,32 @@
 This is the compute_vm metricset of the module azure.
+
+This metricset allows users to retrieve all metrics from specified virtual machines.
+
+[float]
+=== Metricset-specific configuration notes
+
+`refresh_list_interval`
+Resources will be retrieved at each fetch call (`period` interval), this means a number of Azure REST calls will be executed each time.
+This will be helpful if the azure users will be adding/removing resources that could match the configuration options so they will not added/removed to the list.
+To reduce on the number of API calls we are executing to retrieve the resources each time, users can configure this setting and make sure the list or resources will not be refreshed as often.
+This is also beneficial for performance and rate/ cost reasons (https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-manager-request-limits).
+
+`resources` - This will contain all options for identifying resources and configuring the desired metrics
```

Comment:
Be consistent between both options here, this one has a `-` after the option name, but `refresh_list_interval` hasn't it.

Or use the format used in other parts of the documentation, for example in `libbeat/docs/shared-autodiscover.asciidoc`.
```
`host`:: (Optional) Identify the node where {beatname_lc} is running in case it
  cannot be accurately detected, as when running {beatname_lc} in host network
  mode.
`namespace`:: (Optional) Select the namespace from which to collect the
  metadata. If it is not set, the processor collects metadata from all
  namespaces. It is unset by default.
...
```